mne/surface.py

```diff
@@ -1113,3 +1177,46 @@ def mesh_dist(tris, vert):
                           axis=1))
     dist_matrix = csr_matrix((dist, (edges.row, edges.col)), shape=edges.shape)
     return dist_matrix
+
+
+def _load_ascii_surface(filepath, swap=False):
+    """Function for reading triangle definitions from an ascii file.
+
+    Parameters
+    ----------
+    fname_in : str
+        Path to surface ASCII file (ending with '.tri').
+    swap : bool
+        Assume the ASCII file vertex ordering is clockwise instead of
+        counterclockwise.
+
+    Returns
+    -------
+    surf : tuple (nodes, tris)
+        The surface.
```

Comment:
We should probably make this public, call it `read_tri`, and return `rr, tris` just like `read_surface` does